src/components/Pages/Create/CreateProject.js

```diff
@@ -0,0 +1,183 @@
+import React, { useEffect, useState } from 'react'
+import PropTypes from 'prop-types'
+import styled from 'styled-components/macro'
+
+import Header from '../../Header/Header'
+import TagCluster from '../../TagCluster/TagCluster'
+
+export default function Create({ onBack }) {
+  const [tags, setTags] = useState(null)
+  const [image, setImage] = useState()
+  const [newProject, setNewProject] = useState({})
+
+  useEffect(() => {
+    fetch('https://unfinished-api.herokuapp.com/api/tags')
```

Comment:
You should use environment variables to insert the url here. That way you can insert different urls for development and on heroku.

@see https://create-react-app.dev/docs/adding-custom-environment-variables 
and https://devcenter.heroku.com/articles/config-vars#using-the-heroku-dashboard

Reply:
Also: create a function `getTags`, put it in a file `getTags.js` in a folder `src/services` in your project. Import it here.